src/hooks/bridge/Currencies.ts

```diff
@@ -27,11 +29,31 @@ export function useLiFiSwapCurrencies() {
   });
 }
 
+export function useSquidCurrencies() {
+  return useQuery(['squidCurrencies'], async () => {
+    const response = await fetch(`${SQUID_API}/v1/tokens`);
```

Comment:
can we use axios here as here also `src/state/pbridge/hooks.ts` you are using axios

Reply:
I directly used 0xSquid SDK.